CONTRIBUTING.md

```diff
@@ -98,6 +98,16 @@ This path should prioritize:
    5. Contributing to Bevy itself.
 6. Linking to further reading: official examples, `docs.rs` and (very sparingly) source code links.
 
+## Setup
+
+This website is built using [Zola](https://www.getzola.org/): install it and use `zola serve --open` to examine the website as it would appear.
```

Comment:
is `--open` required? I always only used `zola serve`

Reply:
No, it's not. It's a bit more comfortable IMO, but eh, I'll cut it.

Reply:
```suggestion
This website is built using [Zola](https://www.getzola.org/): install it and use `zola serve` to examine the website as it would appear.
```